profiles/models.py

```diff
@@ -68,12 +68,12 @@ class Profile(models.Model):
 
 class Activity(models.Model):
     activity_id = models.AutoField(primary_key=True)
```

Comment:
I see that it's not from this PR, but still: if you need such kind of field in your model, it is created automatically by default and should not be declared explicitly.

Reply:
Oleksandra and Diana work on this problem in other PR